styles/styles.css

```diff
@@ -601,14 +601,14 @@ main .section.background-gray {
       display: none;
     }
 
-    .tablet {
-      display: block;
-    }
-
     .desktop {
       display: none;
     }
 
+    .tablet {
```

Comment:
this order is important for blocks with both tablet and desktop added at the same time
Eg: Banner (tablet, desktop) - to make is visible for both tablet and desktop layout